components/approvalTextbox.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { approvaltextboxprop } from '../types/Types';
+import Grid from '@mui/material/Grid';
+import Paper from '@mui/material/Paper';
+import Typography from '@mui/material/Typography';
```

Comment:
Just a note: if we are going to use this typo, we should consider using it on the other pages to make it consistent